aleph/pages/home.en.md

```diff
@@ -4,6 +4,8 @@ title: 'Aleph'
 description: 'Search documents, companies and people'
 html: true
 samples: ["Vladimir Putin", "TeliaSonera"]
+warning_title: "Public access temporarily disabled"
+warning_body: "We have disabled public access to prevent Aleph from being overwhelmed by automated anonymous requests."
```

Comment:
Is this the correct file to edit? I always thought that aleph.occrp.org will use the site-specific contents from in [`/site/aleph.occrp.org/pages/home.en.md`](https://github.com/alephdata/aleph/blob/main/site/aleph.occrp.org/pages/home.en.md).

Reply:
I'd add a date here for context, it can be frustrating not knowing when this started

Reply:
Ok, I have updated the PR. Thanks for catching the mix up @tillprochaska!